app/controllers/admin/routes.py

```diff
@@ -147,7 +148,12 @@ def createEvent(templateid, programid=None):
 @admin_bp.route('/eventsList/<eventId>/view', methods=['GET'])
 @admin_bp.route('/eventsList/<eventId>/edit', methods=['GET','POST'])
 def eventDisplay(eventId):
-    # Validate given URL
+    viewCount = EventView.select().where(EventView.event == eventId).count() # count the event views 
```

Comment:
Instead of having a trailing comment why not just change the name of the variable so it is clear by just reading its name.